components/server/src/user/user-service.ts

```diff
@@ -72,6 +72,10 @@ export class UserService {
             // blocked = if user already blocked OR is not allowed to pass
             newUser.blocked = newUser.blocked || !canPass;
         }
+        if (newUser.additionalData) {
+            // When a user is created, it does not have `additionalData.profile` set, so it's ok to rewrite it here.
+            // newUser.additionalData.profile = { acceptedPrivacyPolicyDate: new Date().toISOString() };
```

Comment:
Did you mean to uncomment this or should we remove? I wonder if we need to present the privacy policy somewhere during signup if we don't already so that we could set it automatically like this?

Reply:
Meant to uncomment 🙈. @selfcontained we have this awesome little catch on our login pages which I believe allows us to do just that. 
<img width="821" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/3746eee6-8fc8-47af-851a-ad7cd8147814">